.ci/ci.yml

```diff
@@ -39,7 +39,7 @@ jobs:
       clang:
         CC: "clang-4.0"
         CXX: "clang++-4.0"
-        cmakeOptions: ''
+        cmakeOptions: '-DCLANG_DEFAULT_CXX_STDLIB=libc++'
```

Comment:
this is just a hack... just saying... plz dont ever merge it.

Reply:
yup just checking if this is the issue